src/main.rs

```diff
@@ -182,6 +182,12 @@ fn main() {
         .build()
     {
         Ok(re) => walk::scan(&dir_vec, Arc::new(re), Arc::new(config)),
-        Err(err) => error(err.description()),
+        Err(err) => error(
+            format!(
+                "{} \n(Hint: Try the --fixed-strings \
+                 flag to search for a literal string.)",
```

Comment:
Maybe: "Hint: You can use the '--fixed-strings' option to search for a literal string instead of a regular expression"?

Reply:
@sharkdp right, it's more meaningful, will push this code change.